File: app/src/main/java/com/chase/alcandroidchallenge1/NavigationUtils.java
package com.chase.alcandroidchallenge1;

import android.content.Context;
import android.content.Intent;

public class NavigationUtils {

    public static void navigateToProfile(Context context) {
        Intent intent = new Intent(context, ProfileActivity.class);
        context.startActivity(intent);
    }

    public static void navigateToAbout(Context context) {
        Intent intent = new Intent(context, AboutActivity.class);
        context.startActivity(intent);
    }
}

File: app/src/main/java/com/chase/alcandroidchallenge1/NavigationUtils.kt
package com.chase.alcandroidchallenge1

import android.content.Context
import android.content.Intent

object NavigationUtils {

    fun navigateToProfileActivity(context: Context) {
        val intent = Intent(context, ProfileActivity::class.java)
        context.startActivity(intent)
    }

    fun navigateToAboutActivity(context: Context) {
        val intent = Intent(context, AboutActivity::class.java)
        context.startActivity(intent)
    }
}